docs/azure/migration/app-service.md

```diff
@@ -6,7 +6,7 @@ ms.date: 08/11/2018
 ---
 # Migrate your .NET web app or service to Azure App Service
 
-[App Service](https://docs.microsoft.com/azure/app-service/overview) is a fully managed compute platform service that's optimized for hosting scalable websites and web applications. This article provides information on how to lift-and-shift an existing application to Azure App Service, modifications to consider, and additional resources for [moving to the cloud](https://azure.microsoft.com/migration/web-applications/). Most ASP.NET websites (Webforms, MVC) and services (Web API, WCF) can move directly to Azure App Service with no changes. Some may need minor changes while others may need some refactoring.
+[App Service](/azure/app-service/overview) is a fully managed compute platform service that's optimized for hosting scalable websites and web applications. This article provides information on how to lift-and-shift an existing application to Azure App Service, modifications to consider, and additional resources for [moving to the cloud](https://azure.microsoft.com/migration/web-applications/). Most ASP.NET websites (Webforms, MVC) and services (Web API, WCF) can move directly to Azure App Service with no changes. Some may need minor changes while others may need some refactoring.
```

Comment:
Are we supposed to be differentiating "ASP.NET" and "ASP.NET Core"

Reply:
In context, since we mention webforms, I think it should just be "ASP.NET" in this case.